std/stdio.d

```diff
@@ -2808,6 +2808,90 @@ is empty, throws an `Exception`. In case of an I/O error throws
         assert(i == witness.length);
     }
 
+/**
+Like `asCharsRange` but returns a range o `ubyte`s, not `char`s.
+*/
+    auto asUbytesRange(size_t bufferSize = 4096)
+    {
+        assert(bufferSize > 0);
```

Comment:
There is already a check in `byChunk`:

https://github.com/dlang/phobos/blob/master/std/stdio.d#L2669

Reply:
But should it be an exception or assert? Maybe different in `byChunk` and `asUbytesRange`? Not a very easy question.